src/_content/posts/guide-to-america.md

```diff
@@ -0,0 +1,72 @@
+
+- Restaurants: 20% for everything except for the big-name fast food restaurants (e.g. McDonald's, Wendy's, Burger King)
+   - Even some self-serve kiosk ordering places will try to ask for a tip! This money will theoretically go to the staff preparing and/or delivering your food, but it definitely feels odd.
+- Coffee shops: $1-2 per cup of coffee
+- Bars: $1-2 per beer/glass of wine, $2-4 per cocktail, 20% on bottles of wine
+- Rideshare and taxis: 20%
+- Hotel room cleaning: ~$5/day (leave it on your pillow, weirdly)
+- Hotel valet parking: ~$3 per time they retrieve your car
+
+## Identification/Purchasing Alcohol
+
+To purchase alcohol or enter places that predominantly serve alcohol, there's a very good chance you'll be required to present identification. You _may_ be able to get away with a national ID card, but your safest option is your passport. It needs to be the original; photocopies are not accepted.
+
+## Sales Tax
+
+Many countries have some form of value added tax or goods and services tax rolled into the retail price that you pay, so a bottle of water that's listed as costing $2 is exactly what you  pay. In the US, this is very much _not_ the case. Instead, each state charges its own sales tax, and in many states, counties and municipalities add their own tax on top of that. Some places charge different tax rates for different classes of goods, such as a discount for food.
+
+In Durham County, NC, the sales tax is a flat 7.5%. The city of Durham does not add its own sales tax. For example, you will pay $2.15 for the previous hypothetical $2 bottle of water.
+
+## Payments
+
+Almost every venue nowadays takes Visa, MasterCard, American Express, and usually Discover. Make sure you notify your bank before you are traveling that you are going to the United States so you don't set off any fraud alerts. In the US, this requires calling the customer service number printed on the back of the card, but overseas banks may differ.
+
+## Mobile Phones
+
+The United States has 3 main wireless telecommunications providers: Verizon, AT&T, and T-Mobile. Now that Verizon has retired its 3G service, you should be able to bring nearly any phone that supports Voice over LTE (all 5G phones automatically do) and have it work in the US via roaming. However, that can be super expensive depending on your plan back home.
+
+If you want to get a temporary US plan, you have numerous options. The big three carriers are all post-paid, typically requiring long contract terms which make them unsuitable for a quick visit. However, there are numerous providers called MVNOs (mobile virtual network operators) that offer prepaid service, including:
+
+- Boost Mobile
+- Cricket
+- Visible
+- Google Fi
+- Tracfone
+- Mint Mobile
+
+Many of these are able to offer a "bring your own phone" plan where you would have to get a SIM card, likely having it shipped to your hotel and then doing the swap yourself after you land in the US.
```

Comment:
If you prefer to have sim cards, its preferred to get one from the airport that is pre-activated it can be on your final destination or if you are transiting to a different state in the US you can get it on transit too.
Another option would be to get an E-Sim if your phone supports it, some of the common one's include:
- [Airalo](https://airalo.pxf.io/PyjQ1z)
- [Holafly](https://esim.holafly.com/?ref=y2u0ntk&discount=FREAKINGNOMADS&utm_source=freakingnomads&utm_medium=affiliation)
- [Nomad](https://www.getnomad.app/?rfsn=7460813.661b80&ref=freakingnomads.com)
- [aloSIM](https://alosim.com/?_ef_transaction_id=&oid=9&affid=70)
- . [Jetpac](https://circleslife.pxf.io/DKmX0o)